lib/matplotlib/colorbar.py

```diff
@@ -1118,6 +1118,8 @@ def make_axes(parents, location=None, orientation=None, fraction=0.15,
     pad = kw.pop('pad', loc_settings['pad'])
 
     # turn parents into a list if it is not already
+    if isinstance(parents, np.ndarray):
```

Comment:
I would replace this and the lines below by
```
parents = np.atleast_1d(parents).ravel().tolist()
```
(there will never be enough axes for this to be a performance issue...)